hasjob/templates/detail.html

```diff
@@ -1,7 +1,7 @@
 {% extends "sheet.html" %}
 {%- from "baseframe/forms.html" import ajaxform %}
 {%- from "macros.html" import jobheader, stickie %}
-{% block title %}{{ headline }}{% endblock %}
+{% block title %}{{ headline }}, at {{ post.company_name }} in {{ post.location }}{% endblock %}
```

Comment:
I'd avoid English words like "at" and "in". Use punctuation to separate instead. Ideally `/`.